projects/sbb-esta/angular-public/src/lib/usermenu/_usermenu.scss

```diff
@@ -50,7 +58,6 @@ $borderActive5K: solid toPx(4 * $scalingFactor5k) $sbbColorGrey;
   align-items: center;
   margin-left: 3px;
   margin-top: 3px;
-  padding-left: 16px;
```

Comment:
Removing this will cause the icon to be to close to the border when the dropdown is open.
It might make sense to move this to the `usermenuLoggedInActive` mixin.
Be aware that this is potentially a ui breaking change for consumers. Be very carefull with these types of changes.

Reply:
This actually solve the issue that was highlighted, when the user click on login everything moves to the right because of this padding-left: 16px. So let me know if is an expected behavior or not.

Reply:
The specification (https://digital.sbb.ch/de/websites/components/usermenu) specifies no padding  on collapsed state and padding on dropdown state.
Simply removing the padding will fix the issue in collapsed state but cause a problem in the dropdown state.